pkg/archive/archive.go

```diff
@@ -633,6 +647,11 @@ func TarWithOptions(srcPath string, options *TarOptions) (io.ReadCloser, error)
 				}
 
 				if err := ta.addTarFile(filePath, relFilePath); err != nil {
+					if err == errSkipFilePermissions {
+						// GH18120. Note next line is deliberately NOT logrus or %q
+						fmt.Printf("Skipping '%s' - Permission was denied\n", filePath)
```

Comment:
OK, removed it. Getting it back to the caller is extremely problematic unless I change the interface to TarWithOptions, which may very likely be called from components outside of the engine. I only added it in the first place (compared with the original PR) to give a little user feedback, but it's really not essential.